ibm/mas_devops/roles/cp4d_db2wh/tasks/setup_norootsquash.yml

```diff
@@ -0,0 +1,51 @@
+# If you are using IBM® Cloud File Storage (ibmc-file-gold-gid storage class) for Red Hat® OpenShift® Kubernetes Service with Db2®, 
+# you must use the cp.icr.io/cp/cpd/norootsquash:3.0-amd64 image to set no_root_squash.
+
+# 1. Set 'cpregistrysecret' secret content
+- name: Set 'cpregistrysecret' secret content
+  vars:
+    entitledAuthStr: "{{ cpd_registry_user }}:{{ cpd_entitlement_key }}"
+    entitledAuth: "{{ entitledAuthStr | b64encode }}"
+    content:
+      - "{\"auths\":{\"{{ cpd_registry }}/cp/cpd\":{\"username\":\"{{ cpd_registry_user }}\",\"password\":\"{{ cpd_entitlement_key }}\",\"email\":\"{{ cpd_registry_user }}\",\"auth\":\"{{ entitledAuth }}\"}"
+      - "}"
+      - "}"
+  set_fact:
+    new_secret: "{{ content | join('') }}"
+
+# 2. Generate 'cpregistrysecret' secret
+- name: "Generate 'cpregistrysecret' secret"
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: v1
+      kind: Secret
+      type: kubernetes.io/dockerconfigjson
+      metadata:
+        name: cpregistrysecret
+        namespace: kube-system
+      data:
+        .dockerconfigjson: "{{ new_secret | to_json | b64encode }}"
+  register: secretUpdateResult
+
+# 3. Create DeamonSet
+- name: Create 'norootsquash' DeamonSet
+  community.kubernetes.k8s:
+    definition: "{{ lookup('template', 'templates/norootsquash_deamonset.yml') }}"
```

Comment:
This template doesn't appear to exist: https://github.com/ibm-mas/ansible-devops/tree/norootsquash/ibm/mas_devops/roles/cp4d_db2wh/templates